extensions/spark/kyuubi-spark-authz/src/main/resources/table_command_spec.json

```diff
@@ -1244,7 +1244,7 @@
     "fieldExtractor" : "LogicalPlanQueryExtractor"
   } ]
 }, {
-  "classname" : "org.apache.spark.sql.execution.datasources.InsertIntoHiveDirCommand",
```

Comment:
Can we make sure where or which spark version it comes from before removing support for the existed command removed from spec?

Reply:
have checked spark version about 2.4, 3.1 and 3.2, i think it's a mistake of writing a wrong classname.

Reply:
OK, thanks for the check.